tutorials/notebooks/synphot-model-spec/synphot-model-spec.ipynb

```diff
@@ -0,0 +1,998 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "trans_waves, transmission = skymodel.get_atmospheric_transmittance(airmass=1.5)\n",
+    "\n",
+    "atmosphere = SpectralElement(Empirical1D,\n",
+    "                             points=trans_waves,\n",
+    "                             lookup_table=transmission)\n",
+    "atmosphere.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"qe\"></a>\n",
+    "
```

Comment:
Hmm... Did we discuss why it was 51% difference? Seems a little high.

Reply:
@pllim I believe we did discuss it - since we are comparing the model counts to real counts Brett took from the ground on a specific night, I think the consensus we reached was that it likely has to do with whatever the observing conditions were that night, especially since the error of the other ground-based count prediction (of HAT-P-11) agrees with the predicted space-based count errors (all less than %15).

Reply:
I guess we did, but I can't say I remember. Maybe @bmorris3 does.